### PR TITLE
Download formplayer jar from jenkins

### PR DESCRIPTION
**Pull request by benrudolph**
@wpride @emord @esoergel this should help speed up deploys. it downloads most recent formplayer jar from jenkins instead of building it every time. going to test it out on staging

### CODE REVIEW

File: fab/fabfile.py [hunk 1]
```diff
@@ -765,10 +765,10 @@ def copy_formplayer_properties():
 @task
 @roles(ROLES_TOUCHFORMS)
 def build_formplayer():
-    spring_dir = '{}/{}'.format(env.code_root, 'submodules/formplayer')
-    with shell_env(JAVA_HOME='/usr/lib/jvm/jdk1.7.0'):
-        with cd(spring_dir):
-            sudo('./gradlew build')
+    build_dir = '{}/{}'.format(env.code_root, 'submodules/formplayer/build/libs')
+    jenkins_formplayer_build_url = 'http://jenkins.dimagi.com/job/formplayer/lastSuccessfulBuild/artifact/build/libs/formplayer.jar'
```
Comment (dimagimon) [inline fab/fabfile.py]: E501 line too long (132 > 115 characters)

**Comment by esoergel** [comment]
:100:

**Comment by wpride** [comment]
very nice

File: fab/fabfile.py (hunk 1 above)
Comment (benrudolph) [inline fab/fabfile.py]: ignoring

**Comment by benrudolph** [comment]
this works on staging. we should merge this soon since it'll help with deploys taking forever

**Pull request merged**